tests/common.py

```diff
@@ -255,6 +256,7 @@ def async_create_task(coroutine):
                 device_registry.async_load(hass),
                 entity_registry.async_load(hass),
                 issue_registry.async_load(hass),
+                template.async_materialize_hass_loader(hass),
```

Comment:
This adds I/O to every mock hass object in every test, while it's never used. We shouldn't load it here. If needed, just insert an empty HassLoader object.

Reply:
Whoops -- 100% right. Will fix.

Reply:
Done.